lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -541,13 +574,52 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain_and_all_template_disks(self, template):
+
+        if self._module.params.get('template') is None:
+            return None
+
+        if self.param('storage_domain') is None:
+            return None
+
+        storage_domain = self.param('storage_domain')
+
+        disk_attachments = (
```

Comment:
I think it's not worth to create the temp var here, maybe just pass the `self._connection.follow_link(template.disk_attachments)` directly in `for`, also it's not strict to use 80 chars/line and I think here the newline makes it less readable, so if you want to use temp var then I think it would be more readeble, to not use new line, but up to you.